tools/src/main/java/org/apache/pdfbox/tools/PDFWithTimeNewRomanText.java

```diff
@@ -21,9 +21,14 @@ public static void main(String[] args) {
             // For writing to a page content stream
             try(PDPageContentStream cs = new PDPageContentStream(pdfDoc, firstPage)){
                 cs.beginText();
-                // setting font family and font size
-                System.out.println("Enter the type of Font");
-                cs.setFont(PDType1Font.TIMES_ROMAN, 15);
+                // setting font family and font size base on use input.
+                System.out.println("Enter 1 for New Times Roman or 2 for Courier ");
+                int input=myObj.nextInt();
+                if (input==1){
+                    cs.setFont(PDType1Font.TIMES_ROMAN, 15);
+                }
+                else
```

Comment:
minor, but it might be good to have only an input of 2 change to courier with an error message for an input that wasn't asked for.